canonix/tests/golden/chained-lambdas.output.nix

```diff
@@ -0,0 +1,2 @@
+a: b: c:
+a+b + c
```

Comment:
this is obviously bad, but that's the point

Reply:
I suppose the point is observing the formatter through the test suite